src/app/pages/public/LivesPage/index.jsx

```diff
@@ -0,0 +1,73 @@
+import BasicLayout from '../../../layouts/BasicLayout';
+import { Table, Tag, Space } from 'antd';
+import { useGetListOfLifeLazyQuery } from '../../../api';
+import { useEffect } from 'react';
+import LoadingElement from '../../../components/LoadingElement';
+
+const columns = [
+    {
+        title: 'First Name',
+        dataIndex: 'firstName',
+        key: 'firstName',
+    },
+    {
+        title: 'Last Name',
+        dataIndex: 'lastName',
+        key: 'lastName',
+    },
+    {
+        title: 'Full Name',
+        dataIndex: 'fullName',
+        key: 'fullName',
+    },
+    {
+        title: 'Birthday',
+        dataIndex: 'birthday',
+        key: 'birthday',
+    },
+    {
+        title: 'Title',
+        dataIndex: 'title',
+        key: 'title',
+    },
+    {
+        title: 'Description',
+        dataIndex: 'description',
+        key: 'description',
+    },
+    {
+        title: 'Hobbies',
+        dataIndex: 'hobbies',
+        key: 'hobbies',
+        render: tags => (
+            <>
+                {tags.map((tag, i) => {
+                    let color = i % 2 === 0 ? 'geekblue' : 'green';
+                    return (
+                        <Tag color={color} key={tag}>
+                            {tag.toUpperCase()}
+                        </Tag>
+                    );
+                })}
+            </>
+        ),
+    },
+];
+
+const LivesPage = () => {
+    const [fetch, { data, loading }] = useGetListOfLifeLazyQuery();
+
+    useEffect(() => {
+        fetch();
+    }, []);
```

Comment:
don't use lazy here

Reply:
@amille44420 Hey Adrian, would you mind telling me any reason for not using lazy Query here?

Reply:
You use lazy if you don't have the required the data needed for your APIs yet.
For this case since you are getting for the list, you can use `useGetListOfLifeQuery` and this will already fetch the data for you